src/net/uds/stream.rs

```diff
@@ -0,0 +1,159 @@
+use crate::event::Source;
+use crate::unix::SourceFd;
+use crate::{sys, Interests, Registry, Token};
+
+use std::io::{self, IoSlice, IoSliceMut};
+use std::net::Shutdown;
+use std::os::unix::io::{AsRawFd, FromRawFd, IntoRawFd, RawFd};
+use std::os::unix::net::SocketAddr;
+use std::path::Path;
+
+/// A non-blocking Unix stream socket.
+#[derive(Debug)]
+pub struct UnixStream {
+    std: std::os::unix::net::UnixStream,
```

Comment:
Same as for `UnixDatagram` and `UnixListener`:
* `std` -> `sys`,
* `std::os::unix::net::UnixListener` -> `net::UnixListener`, and
* add `SelectorId`.

Reply:
Answered below in a PR comment